sound/soc/sof/intel/bdw.c

```diff
@@ -489,13 +489,12 @@ static int bdw_get_reply(struct snd_sof_dev *sdev, struct snd_sof_ipc_msg *msg)
 	} else {
 		/* reply correct size ? */
 		if (reply.hdr.size != msg->reply_size) {
```

Comment:
So, besides removing the else clause your patch also modifies the format for dev_err.  I think that should be the subject of another patch.